src/irmin/tree.ml

```diff
@@ -1308,6 +1308,7 @@ module Make (P : Private.S) = struct
     | `Contents (k, m) -> `Contents (Contents.of_hash repo k, m)
 
   let export ?clear repo contents_t node_t n =
+    let non_impacting = false in
```

Comment:
`non_impacting = false` reads to me that it is actually impacting :P Perhaps:

```ocaml
 let cache = 
   (* This choice of [cache] flag has no impact, since we either immediately clear the
      corresponding cache or are certain that the it is already filled. *)
   false
 in

Reply:
Much better :D

Reply:
I've changed the definition to 
```
      match clear with
      | Some true | None ->
          false
      | Some false -> true
```
It does matter when `clear` is `false`